webapp/src/app/home/leaderboard/filter/repository/repository.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, computed, effect, signal } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { Router, RouterLink } from '@angular/router';
+import { BrnSelectModule } from '@spartan-ng/ui-select-brain';
+import { HlmSelectModule } from '@spartan-ng/ui-select-helm';
+import { HlmLabelModule } from '@spartan-ng/ui-label-helm';
+
+export const repositoryNames = [
+  'ls1intum/Artemis',
```

Comment:
Can you move this into the environment? Or actually, I think it would be better if we get the available repositories from the server 🤔

Reply:
I thought about this as well, but does it actually make sense to have individual client-sided requests for small UI elements? Do we plan on changing the repository-list often?

I remember we talked about this when implementing the timeframe-filter and I'm personally not convinced it actually make sense. In NextJS this could be solved via Server-Components (SSR), but I don't think there is an equivalent in our current stack.

Maybe we later on create a config-endpoint to get all of these UI content data points in one request (similar to how i18n works)?

Reply:
Just created the `GET /meta`-endpoint according to what we discussed yesterday. For now it only returns the repositories env-var, but we can recycle it for the adjustable configs, which I'll work on after this.